classes/DirtEventTypes.sc

```diff
@@ -38,18 +38,18 @@ DirtEventTypes {
 			play: #{
 
 				var freq, lag, sustain, func;
-				var args, midiout, hasGate, midicmd, latency;
+				var args, midiout, hasGate, midicmd, latency, chan;
+				var nrpnMSB, nrpnLSB, valMSB, valLSB;
 				midicmd = ~midicmd;
-
 				if(midicmd.isNil) {
 					if(~ccn.notNil) { midicmd = \control; ~ctlNum = ~ccn };
 					if(~ccv.notNil) { midicmd = \control; ~control = ~ccv };
 					if(~nrpn.notNil) {
 						midicmd = \control;
-						~nrpnLSB = ~nrpn % 128;
-                        ~nrpnMSB = (~nrpn - ~nrpnLSB) / 128;
-						~valLSB = ~val % 128;
-                        ~valMSB = (~val - ~valLSB) / 128;
+						nrpnLSB = ~nrpn % 128;
+                        nrpnMSB = (~nrpn - nrpnLSB) / 128;
```

Comment:
could you check the formatting? Mabe you need to use be tabs, not spaces.

Reply:
Is it any better now?

Reply:
no, not yet! Not sure what it is. You can hit tab in the IDE (and show invisible characters to double check that there are tabs not spaces)

Reply:
Ok sorry, sorted now. ("Show spaces and tabs" seems to have no effect on my 3.10.3 supercollider compiled under linux.)